packages/storybook/stories/ChatComposite/snippets/CustomDataModelExampleContainer.snippet.tsx

```diff
@@ -65,13 +66,29 @@ export const CustomDataModelExampleContainer = (props: CustomDataModelExampleCon
       }
     });
 
+  // Custom Menu Item Callback for Participant List
+  const onFetchParticipantMenuItems: ParticipantMenuItemsCallback = (participantId, userId, defaultMenuItems) => {
```

Comment:
nit: same here on param types and console logs

Reply:
Seems redundant given that we have already provided a type to onFetchParticipantMenuItems